package.json

```diff
@@ -732,7 +749,7 @@
         "command": "vscode-couchbase.runSearch",
         "key": "ctrl+shift+e",
         "mac": "cmd+shift+e",
-        "when": "(editorLangId == cbs.json || resourceFilename =~ /.cbs.json$/) && !isKVCluster"
+        "when": "editorLangId == json && resourceFilename =~ /\\.cbs\\.json$/ && !isKVCluster && isSearchEnabled"
```

Comment:
QQ: Do we use query to search or just the SDK?
If we directly use SDK, then probably we don't require isKVCluster check